storybookjs/storybook
# Addon-docs: Bugfixes for new on-demand architecture

Issue: N/A

## What I did

Docs fixes for on-demand story store

- [x] Got `official-storybook` working on new branch
- [x] Fix unbalanced React hooks error in ArgsTable
- [x] Fix MDX compiler to emit `mdxComponentAnnotations` (renamed property)
- [x] Fix Source block to show snippets from arbitrary story IDs
- [x] Fix modern inline rendering with absolutely positioned elements (a little hacky)
- [x] Fix how named exports get converted to names in stories list server
- [x] Added simple e2e test for docs tab
- [x] Fix web-components docs tab test (pre-existing bug in addon-docs/web-components)

self-merging @tmeasday -- if you have any comments happy to follow up

## How to test

See the `Addons/Docs` stories in `official-storybook` and manually compare them to `next`


If your answer is yes to any of these, please make sure to include it in your PR.
